### ISSUE TITLE
No connection to nextcloud from my Nextcloud-Desktop when plugin End to End Encryption is enabled

### ISSUE THREAD

**Issue by JOduMonT** [nextcloud/end_to_end_encryption#211]
related to: https://help.nextcloud.com/t/no-connection-to-nextcloud-at-nextcloud-mydomain-com-in-single-desktop-app-works-fine-in-about-5-others/55437/8

I've got this issue too: 
- **Nextcloud server** version 20.0.1 (PHP 7.4.12 ) with the plugin **End-to-End Encryption** version 1.6.2. *I don't see anything in the logs related to any warning, error, info about an issue.*

- My Nextcloud desktop version 2.6 under Ubuntu version 20.04 LTS was showing the message **No connection to Nextcloud** until I disable the plugin and simply restart my Nextcloud-desktop application.

[quote="gas85, post:8, topic:55437"]
I know how it should work, but did not use it since long time. Try to open a ticket direct on github encryption app:

![|16x16](https://github.githubassets.com/favicon.ico) [GitHub ](https://github.com/nextcloud/end_to_end_encryption/issues)

![|60x60](https://avatars2.githubusercontent.com/u/19211038?s=400&amp;v=4)

### [nextcloud/end_to_end_encryption ](https://github.com/nextcloud/end_to_end_encryption/issues)

:closed_lock_with_key: Server API to support End-to-End Encryption - nextcloud/end_to_end_encryption
[/quote]

**Comment by sireebob** [comment]
i had this issue and needed to upgrade to nextcloud desktop 3.x series, which is not available in the ubuntu repos. i have been using the appimage from the nextcloud website. this still seems like a bug to me though.

**Comment by mergey** [comment]
got the same issue

E2E worked perfektly before nextcloud desktop 3.x was published
a roll back don't worked for me

sometimes waiting about 30min to sync a 1KB txt-file
but most the time there is no sync at all with the desktop app

no matter if or if not there is an E2E encrypted folder

deactivating the E2E app was the only workaround for me

**Comment by relaxalien** [comment]
> got the same issue
> 
> E2E worked perfektly before nextcloud desktop 3.x was published
> a roll back don't worked for me
> 
> sometimes waiting about 30min to sync a 1KB txt-file
> but most the time there is no sync at all with the desktop app
> 
> no matter if or if not there is an E2E encrypted folder
> 
> deactivating the E2E app was the only workaround for me

same issue here

**Comment by sysoppl** [comment]
Same, 432 Locked error in desktop app

**Comment by hdlineage** [comment]
Enabling E2E slows down the sync process. It hangs at "Preparing to sync" and sometimes takes forever to upload small files. 
I'm currently on Nextcloud 19 and windows client 3.x

**Comment by NukeManDan** [comment]
Same issue here as OP

**Comment by jonathanmmm** [comment]
Is maybe related to https://github.com/nextcloud/desktop/issues/2593 ?

**Comment by rfreytag1** [comment]
Hello,
I was troubleshooting why my client wouldn't sync for hours today and it was indeed this E2EE module causing the issues. Disabled it and everything syncs fine again.
Weirdly enough it only started causing issues when I disabled Server-Side encryption.
Error logs do not show anything at all.

NextCloud Version: 20.0.4
NextCloud Client Version: 3.1.1 on Windows 10

**Comment by beautifulplace** [comment]
I'm having the same problem. It works fine when I disable the End-to-End Encryption app but I would really like to use e2ee.
NextCloud Version: 20.0.5
NextCloud Client Version: Version 2.6.2-1build1 (Ubuntu 20.04)

**Comment by Fixmetal** [comment]
Same issue here:
Nextcloud Server: 20.0.5 (containerized)
Nextcloud Client: 3.1.1 AppImage
OS: Fedora 33

Yesterday I let my PC turned on overnight and NEVER synched. Just disabled (thanks to some random googling) the E2E app and started synching like never before.
I daily use my NX server both for work and personal stuff and I noticed some days ago that it didn't just sync anymore. I must say that *it seemed* (I can't confirm this though) to sync correctly back up until I manually upped files onto my Server (i.e. rsync to the named docker volume + chown).
I'm able to do any troubleshoot if any contributor has questions. Just let me know because I think this is a very promising feature.

**Comment by cmattern** [comment]
Adding some detail here which may prove helpful. I encountered much the same as the other commenters. Sync was working, at some point it stopped but I also don't have an exact tipping point. What I do have to add is that E2E was also adversely impacting a sync via WebDAV to my TrueNAS core server. A few files got through every night but the sync even inevitably failed due to excessive errors such as this one

> 2021/05/02 00:16:55 ERROR : Photos/Life/Years/2014: error reading source directory: couldn't list files: <html>
> <head><title>504 Gateway Time-out</title></head>
> <body bgcolor="white">
> <center><h1>504 Gateway Time-out</h1></center>
> <hr><center>nginx/1.14.0 (Ubuntu)</center>
> </body>

The error/retry loop consumed all available connections and left my NC instance functionally unavailable, contact and calendar sync to my Android timed out, I was unable to log in through the web UI, etc. When I disabled E2E not only was my desktop sync able to run but my WebDAV sync to my TrueNAS core now runs fine as well. Responsiveness of my VM is reduced due to load but nothing times out. The bottom line is that this is not directly related to the desktop client, it just shows up there, and eslewhere.

As the commenter above I am supportive of the app and the contributors efforts and am more than happy to help with troubleshooting and getting this resolved. 

Server:
OS: Ubuntu 18.04.5 LTS
NC: 20.0.9 Stable
E2E: 1.6.2
